mod/fbrowser.php

```diff
@@ -10,6 +10,7 @@
 /**
  * @param App $a
  */
+/// @TODO & is missing or App ?
```

Comment:
Since $a is an object, it always is passed by reference. I suggest adding the class hint to the parameter list like this:

````php
function fbrowser_content(App $a) {
````